requirements/libraries.txt

```diff
@@ -35,6 +35,7 @@ openpyxl==2.5.8
 psycopg2==2.7.3.2
 pyelasticsearch==0.6.1
 python-dateutil==2.7.3
+pypandoc==1.5
```

Comment:
```suggestion
pypandoc==1.5
python-dateutil==2.7.3
```

Reply:
Fixed in fb68748672adc3af6a50c70501c76809190e9982.